pkg/controller/controlplane/valuesprovider.go

```diff
@@ -322,7 +323,21 @@ func (vp *valuesProvider) GetControlPlaneChartValues(
 		}
 	}
 
-	return getControlPlaneChartValues(cpConfig, cp, cluster, secretsReader, checksums, scaledDown)
+	var useCredentialsFile bool
+	cloudprovider := &corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "cloudprovider",
```

Comment:
We can reuse the name from [here](https://github.com/gardener/gardener/blob/a38354a6c3a0ae734d243553d14bed26c2fcbd0d/pkg/apis/core/v1beta1/constants/types_constants.go#L53)